sage_trac/ticket_box.py

```diff
@@ -131,8 +131,17 @@ def merge_link(url=None, class_='positive_review'):
                     tag.a(class_=class_, href=url))
 
         def commits_link(url):
-            return FILTER_BRANCH.append(tag.span(' ')).\
-                    append(tag.a('(Commits)', href=url))
+            links = tag.span(' (')
+            if url is not None:
+                links.append(tag.a('Commits', href=url))
+                links.append(tag.span(', '))
+            github_url = f"https://github.com/sagemath/sagetrac-mirror/compare/develop...{branch}"
+            gitlab_url = f"https://gitlab.com/sagemath/dev/tracmirror/-/compare/develop...{branch}"
```

Comment:
`SyntaxError`.  Trac still runs on Python 2.

Reply:
Should be fixed.